apis/config/v1beta1/configuration_types.go

```diff
@@ -224,3 +228,41 @@ type Integrations struct {
 	//  - "jobset.x-k8s.io/jobset"
 	Frameworks []string `json:"frameworks,omitempty"`
 }
+
+type QueueVisibility struct {
+	// LocalQueues is configuration to expose the information
+	// about the top pending workloads in the local queue.
+	LocalQueues *LocalQueueVisibility `json:"localQueues,omitempty"`
+
+	// ClusterQueues is configuration to expose the information
+	// about the top pending workloads in the cluster queue.
+	ClusterQueues *ClusterQueueVisibility `json:"clusterQueues,omitempty"`
+
+	// UpdateInterval specifies the time interval for updates to the structure
+	// of the top pending workloads in the queues.
+	// Defaults to 5s.
+	// +optional
+	UpdateInterval *metav1.Duration `json:"updateInterval,omitempty"`
```

Comment:
```suggestion
	UpdateIntervalSeconds int32 `json:"updateInterval,omitempty"`
```
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units

Reply:
updated to UpdateIntervalSeconds